f8a_worker/workers/schemas/github_details.py

```diff
@@ -8,12 +8,14 @@
 ROLE_v1_0_2 = "v1-0-2"
 ROLE_v1_0_3 = "v1-0-3"
 ROLE_v1_0_4 = "v1-0-4"
+ROLE_v1_0_5 = "v1-0-5"
 ROLE_v2_0_0 = "v2-0-0"
 ROLE_TITLE = jsl.roles.Var({
     ROLE_v1_0_0: "Github Results v1-0-0",
     ROLE_v1_0_1: "Github Results v1-0-1",
     ROLE_v1_0_3: "Github Results v1-0-3",
     ROLE_v1_0_4: "Github Results v1-0-4",
+    ROLE_v1_0_5: "Github Results v1-0-5",
     ROLE_v2_0_0: "Github Results v2-0-0",
 })
```

Comment:
Why `1-0-5` and not `2-0-1` ? Looks like @msrb added the `2-0-0` with 5c46e8b, but didn't bump the used schema version in `GithubTask`. @msrb do you remember whether it was on purpose or by mistake ?

Reply:
It was a mistake.